lib/upload/azure/createReverMediaObject.js

```diff
@@ -33,6 +33,7 @@ function buildData(args) {
     originalFileName: args?.file?.name ?? args?.fileName,
     fileExtension: getFileExtension(args),
     bytes: args?.file?.size ?? 0,
+    isPublic: Boolean(args?.isPublic),
```

Comment:
This payload is sent to Rever API in order to create a Rever Media Object after uploading the file to a third-party storage service like Azure.